application/pom.xml

```diff
@@ -89,5 +89,10 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>io.dropwizard</groupId>
+        <artifactId>dropwizard-auth</artifactId>
+        <version>0.8.4</version>
```

Comment:
Since this isn't the parent level pom, you don't need this version number here. It's important that it's not here so that the parent pom can tell this one the correct version number to use and we don't have two different versions in the same project.